core/clingutils/res/TClingUtils.h

```diff
@@ -283,7 +283,7 @@ class AnnotatedRecordDecl {
    const std::string &GetDemangledTypeInfo() const { return fDemangledTypeInfo; }
    bool HasClassVersion() const { return fRequestedVersionNumber >=0 ; }
    bool RequestStreamerInfo() const {
-      // Equivalent to CINT's cl.RootFlag() & G__USEBYTECOUNT
+      // Equivalent to CLING's cl.RootFlag() & G__USEBYTECOUNT
```

Comment:
Cling does not have a `cl.RootFlag`   ... 
```suggestion
      // Equivalent to CINT's cl.RootFlag() & G__USEBYTECOUNT
```